packages/node/src/utils.ts

```diff
@@ -21,29 +21,23 @@ export function hashOfOrderedPublicIdentifiers(addresses: Address[]): string {
 }
 
 /**
+ * Finds a StateChannel based on two xpubs in a store.
  *
- * @param myIdentifier
- * @param peerAddress Peer Address could either be an intermediary or a
- *        `respondingAddress` which is the targeted peer in a Virtual AppInstance
- *        operation.
- * @param store
+ * @param myXpub - first xpub
+ * @param theirXpub - second xpub
+ * @param store - store to search within
  */
-export async function getChannelFromPeerAddress(
-  myIdentifier: string,
-  peerAddress: string,
+export async function getStateChannelBasedOnTwoXpubs(
```

Comment:
`BasedOn` is not a connector used in function names; `getStateChannelWithOwners` is better